easybuild/easyconfigs/t/timm/timm-1.0.8-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,136 @@
+    {'proc-macro2-1.0.80.tar.gz': 'a56dea16b0a29e94408b9aa5e2940a4eedbd128a1ba20e8f7ae60fd3d465af0e'},
+    {'pyo3-0.21.1.tar.gz': 'a7a8b1990bd018761768d5e608a13df8bd1ac5f678456e0f301bb93e5f3ea16b'},
+    {'pyo3-build-config-0.21.1.tar.gz': '650dca34d463b6cdbdb02b1d71bfd6eb6b6816afc708faebb3bac1380ff4aef7'},
+    {'pyo3-ffi-0.21.1.tar.gz': '09a7da8fc04a8a2084909b59f29e1b8474decac98b951d77b80b26dc45f046ad'},
+    {'pyo3-macros-0.21.1.tar.gz': '4b8a199fce11ebb28e3569387228836ea98110e43a804a530a9fd83ade36d513'},
+    {'pyo3-macros-backend-0.21.1.tar.gz': '93fbbfd7eb553d10036513cb122b888dcd362a945a00b06c165f2ab480d4cc3b'},
+    {'quote-1.0.36.tar.gz': '0fa76aaf39101c457836aec0ce2316dbdc3ab723cdda1c6bd4e6ad4208acaca7'},
+    {'redox_syscall-0.4.1.tar.gz': '4722d768eff46b75989dd134e5c353f0d6296e5aaa3132e776cbdb56be7731aa'},
+    {'ryu-1.0.17.tar.gz': 'e86697c916019a8588c99b5fac3cead74ec0b4b819707a682fd4d23fa0ce1ba1'},
+    {'scopeguard-1.2.0.tar.gz': '94143f37725109f92c262ed2cf5e59bce7498c01bcc1502d7b9afe439a4e9f49'},
+    {'serde-1.0.197.tar.gz': '3fb1c873e1b9b056a4dc4c0c198b24c3ffa059243875552b2bd0933b1aee4ce2'},
+    {'serde_derive-1.0.197.tar.gz': '7eb0b34b42edc17f6b7cac84a52a1c5f0e1bb2227e997ca9011ea3dd34e8610b'},
+    {'serde_json-1.0.115.tar.gz': '12dc5c46daa8e9fdf4f5e71b6cf9a53f2487da0e86e55808e2d35539666497dd'},
+    {'smallvec-1.13.2.tar.gz': '3c5e1a9a646d36c3599cd173a41282daf47c44583ad367b8e6837255952e5c67'},
+    {'syn-2.0.59.tar.gz': '4a6531ffc7b071655e4ce2e04bd464c4830bb585a61cabb96cf808f05172615a'},
+    {'target-lexicon-0.12.14.tar.gz': 'e1fc403891a21bcfb7c37834ba66a547a8f402146eba7265b5a6d88059c9ff2f'},
+    {'unicode-ident-1.0.12.tar.gz': '3354b9ac3fae1ff6755cb6db53683adb661634f67557942dea4facebec0fee4b'},
+    {'unindent-0.2.3.tar.gz': 'c7de7d73e1754487cb58364ee906a499937a0dfabd86bcb980fa99ec8c8fa2ce'},
+    {'windows-targets-0.48.5.tar.gz': '9a2fa6e2155d7247be68c096456083145c183cbbbc2764150dda45a87197940c'},
+    {'windows_aarch64_gnullvm-0.48.5.tar.gz': '2b38e32f0abccf9987a4e3079dfb67dcd799fb61361e53e2882c3cbaf0d905d8'},
+    {'windows_aarch64_msvc-0.48.5.tar.gz': 'dc35310971f3b2dbbf3f0690a219f40e2d9afcf64f9ab7cc1be722937c26b4bc'},
+    {'windows_i686_gnu-0.48.5.tar.gz': 'a75915e7def60c94dcef72200b9a8e58e5091744960da64ec734a6c6e9b3743e'},
+    {'windows_i686_msvc-0.48.5.tar.gz': '8f55c233f70c4b27f66c523580f78f1004e8b5a8b659e05a4eb49d4166cca406'},
+    {'windows_x86_64_gnu-0.48.5.tar.gz': '53d40abd2583d23e4718fddf1ebec84dbff8381c07cae67ff7768bbf19c6718e'},
+    {'windows_x86_64_gnullvm-0.48.5.tar.gz': '0b7b52767868a23d5bab768e390dc5f5c55825b6d30b86c844ff2dc7414044cc'},
+    {'windows_x86_64_msvc-0.48.5.tar.gz': 'ed94fce61571a4006852b7389a063ab983c02eb1bb37b47f8272ce92d06d9538'},
+]
+
+use_pip = True
+
+exts_list = [
+    ('huggingface_hub', '0.24.5', {
+        'checksums': ['7b45d6744dd53ce9cbf9880957de00e9d10a9ae837f1c9b7255fc8fa4e8264f3'],
+    }),
+    ('safetensors', '0.4.3', {
```

Comment:
there are Safetensors in EB: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/s/Safetensors/Safetensors-0.4.3-gfbf-2023a.eb

Reply:
Oh in that case, it's definitely better to use that as a dependency and clean up this easyconfig to not use `crates` at all... :+1: